fms/testing/_internal/model_test_suite.py

```diff
@@ -133,6 +138,35 @@ def test_config_passed_to_model(self, model, config):
         assert model.get_config().as_dict() == config.as_dict()
 
 
+class ModelCompileTestSuite(ModelFixtureMixin):
+    """A set of tests associated with compilation of fms models"""
+
+    @property
+    @abc.abstractmethod
+    def _get_signature_params(self) -> Union[int, List[str]]:
+        """the value to pass into params in get_signature function for this model
+
+        Returns
+        -------
+        Union[int, List[str]]
+            the params to set to the default tensor value (inp) in get_signature. If an integer, will use *args, if a
+            list, will use **kwargs
+        """
+        pass
+
+    @pytest.mark.skipif(
+        platform.system() != "Linux",
+        reason=f"pytorch compile is stable on Linux, skipping as current platform is {platform.platform()}",
+    )
+    def test_model_compile_no_graph_breaks(self, model):
+        """Test that an FMS model is compilable without graph breaks"""
+        try:
+            compiled_model = torch.compile(model, fullgraph=True)
+            get_signature(compiled_model, params=self._get_signature_params)
+        except Exception as e:
+            pytest.fail(f"Model compilation failed with the following response: {e}")
```

Comment:
since `e` could be anything should probably just let the test fail instead of trying to catch here